doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -817,18 +853,15 @@ Testing your patch
 Work-in-progress pull request
 +++++++++++++++++++++++++++++
 
-This is a special kind of pull request that is not yet intended to be merged.
-Such PRs are created to take advantage of what the Github PR interface provides,
-such as public comment history and quick links between the WIP PR and other
-related issues and pull requests.  They allow the developer to get early
-feedback on a change.
+A work-in-progress (WIP) PR is a special kind of pull request that is not yet
+intended to be merged.  Such PRs are created to take advantage of what the
+Github PR interface provides, such as public comment history and quick links
+between the WIP PR and other related issues and pull requests.  They allow the
+developer to get early feedback on a change.
 
-The status of the WIP PR should be clearly stated, including what steps need to
-be taken before the PR is ready for final review.
-
-It is generally advisable to "close" such PRs until they are ready for
-final review and testing, as their development can span a large period of time
-and would thus add clutter to the list of open PRs.
+The status of a WIP PR can be stated by using the GitHub "draft PR" feature.
+The PR description should include what steps need to be taken before the PR is
+ready for final review.
```

Comment:
That's a good point. I think it makes sense to utilize the new github feature for this functionality (draft PRs), especially since external contributors can mark their PRs as draft, while labels require a core contributor to mark it. I'll update the text to reflect when the WIP label may be used (if the PR was not opened as a draft, but ends up being a work-in-progress).